client/src/extension.ts

```diff
@@ -24,31 +24,31 @@ export function activate(context: vscode.ExtensionContext) {
       provideTerminalProfile(
         token: vscode.CancellationToken
       ): vscode.ProviderResult<vscode.TerminalProfile> {
+        const which = require("which");
         const path = require("path");
-        const fs = require("fs");
-        const glob = require("glob");
-        const os = require("os");
 
+        const PATH_FROM_ENV = process.env["PATH"];
         const pathsToCheck = [
+          PATH_FROM_ENV,
           // cargo install location
-          "~/.cargo/bin/nu",
-          "~/.cargo/bin/nu.exe",
+          (process.env["CARGO_HOME"] || "~/.cargo") + "/bin",
 
           // winget on Windows install location
-          "c:\\program files\\nu\\bin\\nu.exe",
+          "c:\\program files\\nu\\bin",
           // just add a few other drives for fun
-          "d:\\program files\\nu\\bin\\nu.exe",
-          "e:\\program files\\nu\\bin\\nu.exe",
-          "f:\\program files\\nu\\bin\\nu.exe",
+          "d:\\program files\\nu\\bin",
+          "e:\\program files\\nu\\bin",
+          "f:\\program files\\nu\\bin",
 
           // SCOOP:TODO
           // all user installed programs and scoop itself install to
           // c:\users\<user>\scoop\ unless SCOOP env var is set
           // globally installed programs go in
           // c:\programdata\scoop unless SCOOP_GLOBAL env var is set
           // scoop install location
-          "~/scoop/apps/nu/*/nu.exe",
-          "~/scoop/shims/nu.exe",
+          // SCOOP should already set up the correct `PATH` env var
+          //"~/scoop/apps/nu/*/nu.exe",
+          //"~/scoop/shims/nu.exe",
```

Comment:
I'd prefer to leave these scoop folders. I've found that "should be" vs "are" are sometimes different.

Reply:
I disagree. if the user just install `scoop` the "normal" way, it's there anyway. but if the directory is not in `env::PATH`, that's an exceptional case, it's more like the user **intentionally** removed it from their environment (e.g. to avoid conflicting with other installations), and I think we'd better respect the `PATH` environment variable more.

Reply:
ok. That makes sense. I hadn't considered that one would have removed it from their path on purpose. Let's just leave it commented out then.